src/ontology/edits/oeo-physical.omn

```diff
@@ -2792,6 +2792,117 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000220>,
+        <http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030002>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010016>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic hydrogen is hydrogen with synthetic origin. It is usually produced from water applying the water electrolysis process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "synthetic hydrogen"@de
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000220>,
+        <http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030005>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010017>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A synthetic fuel is a fuel produced from one or more portion(s) of matter using electric energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "e-fuel",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "electric fuel",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "synthetic fuel"@de
+    
+    EquivalentTo: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000173>
+         and (<http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030005>)
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/RO_0000091> some <http://openenergy-platform.org/ontology/oeo/OEO_00000097>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010018>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic methane is synthetic fuel produced from water and carbon dioxide using electric energy applying the power-to-gas technology.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/134
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
+        rdfs:label "synthetic methane"@de
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000025>,
+        <http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030005>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00010019>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic liquid fuel is synthetic fuel produced from water and carbon dioxide using electric energy applying the power-to-liquid technology.",
```

Comment:
Same comment as above. I would write:
"Synthetic liquid fuel is synthetic fuel produced from water and carbon dioxide using electric energy applying the electrolysis technology as first step, a methanation reaction as second step and the Fischer-Tropsch process as third step. This concept is know as power-to-liquid."
 the power-to-liquid technology.

Reply:
Same as above. What about: _Synthetic liquid fuel is synthetic fuel produced **in a power-to-liquid system** from water and carbon dioxide using electric energy._?